src/states/node.rs

```diff
@@ -2828,6 +2892,11 @@ impl Node {
         }
     }
 
+    /// Shortcut to get our prefix
+    fn our_prefix(&self) -> &Prefix<XorName> {
```

Comment:
There are quite a few remaining places where `self.peer_mgr.routing_table().our_prefix()` should be replaced by `self.our_prefix()`.